.github/workflows/format.yml

```diff
@@ -18,5 +18,5 @@ jobs:
       - uses: actions/checkout@v1
       - name: Format code
         run: |
-          find src/ -type f -name '*.c' -name '*.c' | xargs clang-format -i
+          find src/ -type f -name '*.c' -not -path "src/thirdparty/*" -not -path "src/GUI/icon_data.c" | xargs clang-format -i
```

Comment:
Well I guess that's a way to do it :joy:

Reply:
Well, we shouldn't touch third party, and that one file is something I'd rather not touch either for now. All the rest is neatly formatted from now on :)

Reply:
We already don't touch third party: https://github.com/CCExtractor/ccextractor/blob/master/src/thirdparty/.clang-format

Reply:
Ah, good remark on the config file for third party. I'll remove that.